skin/frontend/rwd/default/js/msrp_rwd.js

```diff
@@ -12,8 +12,8 @@
  */
 
 Catalog.Map.showHelp = Catalog.Map.showHelp.wrap(function (parent, event) {
-    var helpBox = $('map-popup');
-    var bodyNode = $$('body')[0];
+    var helpBox = document.getElementById('map-popup');
+    var bodyNode = document.getElementsByTagName('body')[0];
```

Comment:
`document.querySelector('body')`?

Reply:
Or just `document.body` should work too?

Reply:
There is warning with PhpStorm about `document.body`. I don't remember exactly which.

Reply:
I approved the PR because he replaced the $ and $$ exactly with the vanilla JavaScript equivalents, which are easy to understand in this case. Obviously other variants can be used, but I like ID and TagName more, especially because they are together. Obviously a personal opinion...

Reply:
Since it’s such a small change I approve any of the options. The straight replacement with JS equivalent would be just fine if we applied it broadly across the codebase, instead of trying to optimize with querySelector — which could be handled on a case by case basis.

Reply:
https://stackoverflow.com/questions/18247289/what-is-the-difference-between-queryselectorall-and-getelementsbytagname

in this case it seems to me there's absolutely no difference